### DISCUSSION TITLE
Support debugging with hardhat

### DISCUSSION THREAD

**Discussion by tennox** [vyperlang/vyper#2362] [Ideas]
## Simple Summary
Make logging available via hardhat's `console.log`
Only really understanding a contract makes it secure. And it will ease development experience a lot!

Read the issue in the hardhat repo about it: https://github.com/nomiclabs/hardhat/issues/1018

**Comment by fubuloubu** [comment]
`console.log` is a particular semantic to how Hardhat has to hook into Solidity and enable this feature. We've been discussing adding more direct support with a `print` statement instead.
